### PR TITLE
NextDNS.NextDNS version 3.0.0

### PR DESCRIPTION
**Pull request by DoodlesEpic**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60809)

### CODE REVIEW

**Comment by DoodlesEpic** [comment]
Just tested the install with a local manifest and everything worked fine, altough my system isn't completelly clean. Since NextDNS 3.0.0 is not using the TAP Driver anymore this should be able to be added to winget.
https://github.com/nextdns/windows/issues/16

**Comment by DoodlesEpic** [comment]
Didn't use this draft PR since this was created with wingetcreate. Should be pretty easy to move the changes to that PR though if mantainers so wish.
https://github.com/microsoft/winget-pkgs/pull/38207

File: manifests/n/NextDNS/NextDNS/3.0.0/NextDNS.NextDNS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NextDNS.NextDNS
+PackageVersion: 3.0.0
+PackageLocale: en-US
+Publisher: NextDNS
+PackageName: NextDNS Client
```
Comment (ItzLevvie) [inline manifests/n/NextDNS/NextDNS/3.0.0/NextDNS.NextDNS.locale.en-US.yaml:8]:
```suggestion
PackageName: NextDNS
```
This must match Control Panel.

File: manifests/n/NextDNS/NextDNS/3.0.0/NextDNS.NextDNS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NextDNS.NextDNS
+PackageVersion: 3.0.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/nextdns/windows/releases/download/v3.0.0/NextDNSSetup-3.0.0.exe
+  InstallerSha256: 7CB7C56113C16D9AF813AFFCEC3B277E9B9BFAFBD4E7F023FB80BF49308DA081
```
Comment (ItzLevvie) [inline manifests/n/NextDNS/NextDNS/3.0.0/NextDNS.NextDNS.installer.yaml:10]:
```suggestion
  InstallerSha256: 7CB7C56113C16D9AF813AFFCEC3B277E9B9BFAFBD4E7F023FB80BF49308DA081
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**